src/fontra/views/editor/panel-reference-font.js

```diff
@@ -649,6 +661,44 @@ export default class ReferenceFontPanel extends Panel {
       []
     );
 
+    const addRemoveButtons = createDomElement("add-remove-buttons");
+    addRemoveButtons.addButtonCallback = async () => {
+      // FIXME Add dialog title translation
+      const dialog = await dialogSetup("Upload", null, [
+        {
+          title: translate("dialog.cancel"),
+          resultValue: "cancel",
+          isCancelButton: true,
+        },
+        {
+          title: translate("dialog.add"),
+          resultValue: "ok",
+          isDefaultButton: true,
+        },
+      ]);
+      const fileInput = input({ type: "file", id: "reference-font-file" }, []);
```

Comment:
```suggestion
      const fileInput = input({ type: "file", id: "reference-font-file", accept: "font/*"}, []);
```

Reply:
Is this enough to make it accept ttf, otf, woff and woff2 and nothing else?